test/fft_utest.cpp

```diff
@@ -140,6 +143,67 @@ class FftTest : public ::testing::Test {
             ASSERT_EQ(ifft_1, v);
         }
     }
+
+    /** Convert a number into a vector of digits padded with zeros
+     *
+     * @param vec destination vector
+     * @param n vector size
+     * @param num number in string representation
+     */
+    void big_num_to_vec(vec::Vector<T>& vec, int n, std::string num)
+    {
+        int i;
+        int len = num.length();
+        for (i = 0; i < len; i++) {
+            vec.set(i, num[len - i - 1] - '0');
+        }
+        for (; i < n; i++) {
```

Comment:
Could use `std::fill(vec.begin() + i, vec.end(), 0); ` no?

Or `std::fill_n(vec.begin() + i, n - i, 0);` if `n` is not the length of the vector.

Reply:
it is a special operation where we embed the string into the vector, we must set the ascii zeroes and pad by zeroes.

Reply:
since we cannot transform the above loop this is weird.

Reply:
> since we cannot transform the above loop this is weird.
Why?

I don't see why keeping the above loop as-is prevent to use more explicit function when it's possible.

Reply:
There's not `vec.begin()`, `vec.end` as `vec` is `quadiron::vec::Vector`, not `std::vector`. We could use `std::fill` or `std::fill_n` but we need to get the array handled by `vec`. 
So I think we can keep it like this.

Reply:
Ha since it's our own array with it weird interface, yeah we can keep it as is…